src/gui/accountmanager.h

```diff
@@ -55,14 +56,15 @@ class AccountManager : public QObject {
 
     /**
      * Return a list of all accounts.
-     * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy of them)
+     * (this is a list of QSharedPointer for internal reasons, one should normally not keep a copy
+     * of them)
```

Comment:
I'd personally vote for not setting it and let the people decide how to line-break, long lines never really annoyed me in the code.